Drop unused debian packages

simondeziel opened the pull request:
**What this PR does / why we need it**:

Remove now unneeded apt packages (apt-transport-https and software-properties-common) from install instructions.

*apt-transport-https* is a virtual package since Ubuntu Bionic as apt
natively supports HTTPS repos since then, see:
https://packages.ubuntu.com/bionic/apt-transport-https

*software-properties-common* was used for the add-apt-repository command
that is no longer used since commit 26cf3d9

**Which issue(s) this PR fixes**:

n/a as it's a drive-by PR.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/grafana/grafana?pullRequest=45173) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/grafana/grafana?pullRequest=45173) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/grafana/grafana?pullRequest=45173) it.</sub>

malcolmholmes commented:
@simondeziel can you explain why these packages are no longer needed? What has changed?

simondeziel commented:
@malcolmholmes sure, I updated the description with the info from the 2 commits. Basically, I found 2 packages that were pulled for historical reasons, but they serve no useful purpose now.

simondeziel commented:
If anyone has time for a review, this is still meaningful IMHO.

simondeziel commented:
ping

torkelo commented:
Sorry, can you rebase this PR ? the debian file got moved in main. Ping me after and I can merge it

simondeziel committed:
apt-transport-https is no longer needed (apt has builtin HTTPS support)

apt-transport-https is a virtual package since Ubuntu Bionic as apt
natively supports HTTPS repos since then, see:
https://packages.ubuntu.com/bionic/apt-transport-https

Signed-off-by: Simon Deziel <simon@sdeziel.info>

simondeziel committed:
Stop pulling software-properties-common (unused)

software-properties-common was used for the add-apt-repository
command that is no longer used since commit 26cf3d9c66d8f1

Signed-off-by: Simon Deziel <simon@sdeziel.info>

simondeziel commented:
@torkelo rebase done.
